src/main/java/frc/team2412/robot/Subsystems/ClimbMotorSubsystem.java

```diff
@@ -0,0 +1,27 @@
+package frc.team2412.robot.Subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.RobotMap;
+
+public class ClimbMotorSubsystem extends SubsystemBase {
+	ClimbMotorSubsystem m_ClimbMotor;
+	// private WPI_TalonSRX m_ClimbMotorSubsystem;
+
+	private WPI_TalonSRX leftMotor = RobotMap.leftClimbMotor;
+	private WPI_TalonSRX rightMotor = RobotMap.rightClimbMotor;

```

Comment:
Hardware shouldnt come from robotmap, but from constructor

Reply:
This isn't in my version of the code. I'll upload everything again and that should fix it.